OracleDatabase/RAC/OracleDNSServer/dockerfiles/latest/Dockerfile

```diff
@@ -0,0 +1,80 @@
+# LICENSE UPL 1.0
+#
+# Copyright (c) 1982-2021 Oracle and/or its affiliates. All rights reserved.
+#
+# ORACLE DOCKERFILES PROJECT
+# --------------------------
+# This is the Dockerfile for Oracle Database 19c RAC Storage Server. This file create DNS Server for RAC. 
+# 
+# HOW TO BUILD THIS IMAGE
+# -----------------------
+# Put all downloaded files in the same directory as this Dockerfile
+# Run: 
+#      $ docker build -t oracle/rac-dns-server:19.3.0. 
+#
+# Pull base image
+# ---------------
+FROM oraclelinux:7-slim
+
+# Maintainer
+# ----------
+MAINTAINER Paramdeep saini <paramdeep.saini@oracle.com>, Sanjay Singh <sanjay.singh@oracle.com>
```

Comment:
The `MAINTAINER` tag has been deprecated for a long time. Please switch to using labels as per the [contribution guide](https://github.com/oracle/docker-images/blob/main/CONTRIBUTING.md).